core/domain/user_jobs_one_off.py

```diff
@@ -81,6 +81,41 @@ def reduce(key, version_and_exp_ids):
                     edited_exploration_ids))
 
 
+class UserAuthModelOneOffJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
What's the plan for integrating with the new auth model? Do we need a feconf flag for conditioning downstream code to depend on UserAuthModel vs. the user settings model for before this migration can be run in production?

Reply:
I think the plan is to use a flag

Reply:
@BenHenning Yes, the flag would be introduced for that conditioning. I wasn't sure where I should create it - now or in a later PR when we would actually use UserAuthModel or its methods.
@BenHenning or @vinitamurthi Pls confirm that just creating a constant in constant.ts file is enough for now? (As we don't have to condition anything based on it right now)

Reply:
I think you can add the flag when you actually use it. This PR is only creating the model so its not necessary to add it here.

Reply:
Thanks for clarifying that!

Reply:
Thanks for clarifying the plan. @sajalasati can you update your PR description to capture these details to make it clearer?